### ISSUE TITLE
Bot Not Working

### ISSUE THREAD

**Issue by Code-Bear** [Tune389/Tap-Titans-2-Progress-Bot#86]
I followed each of the steps and recommendations in the readme and HiroMacro seems to be working fine. I tested with my own recording and it works and I tested with the TapTitans2RandomClicker which seem to work fine but when I start the TapTitans2AdvancedClickBot script it says script started but it doesn't do anything. I have tried restarting Nox, deleting the script and re-adding it, and reviewing all the settings from the readme like 10x.

**Comment by Code-Bear** [comment]
Yes, as I stated above I have checked everything on the readme like 10x. Screen capture mode is on, the delay is set to 0, Nox is set to mobile phone at 480x800.

**Comment by atereshkov** [comment]
try to set delay to 2 secs

**Comment by Code-Bear** [comment]
It was set to 2 seconds by default and was not working with that setting either. I just changed it back to try again but it doesn't help. Still the script says it is running but it never does anything. No auto-tapping, no taps on the ship or pet, no lvling, just its there.

**Comment by DaBest1Evar** [comment]
Did you also delete the #checkapp lines in the script? The script no longer works if you don't the lines:

"CheckApp #checkRunning com.gamehivecorp.taptitans2
	if #checkRunning == 0
		RunApp com.gamehivecorp.taptitans2
		WaitApp com.gamehivecorp.taptitans2
		sleep 10000
	endif"
If it still doesn't work after removing those lines, I don't know what the problem is.

**Comment by Code-Bear** [comment]
I didn't do that. I didn't know that was required. Thank you, is this not in dev anymore or is the dev working on a new release? It appears now the macro is running too fast. The taps to lvl your heroes happen so quick they are registering as long presses or something. You can see the button pressed but it is released as if you dragged your finger off the button before lifting it. Any ideas on that? I have plenty of RAM and a crap GPU but it is enough to play the game without issue.

**Comment by atereshkov** [comment]
Try to use this version https://github.com/Tune389/Tap-Titans-2-Progress-Bot/blob/development/Sample%20HS%20Siphon%20Build

**Comment by Code-Bear** [comment]
The new one is working better than the original but it still seems like it is pressing the buttons too fast. If I make a macro and only put like 50ms between the clicks they don't get clicked just looks like a long press but with 500ms between clicks they all register as clicks.
